modules/acs-central-services-overview.adoc

```diff
@@ -6,15 +6,17 @@
 = Central services
 
 You install Central services on a single cluster.
-These services include three main components, Central, Central DB and Scanner.
+These services include four main components, Central, Central DB, StackRox Scanner, and Scanner V4 (optional).
 
 * *Central*: Central is the {product-title-short} application management interface and services.
 It handles API interactions and user interface ({product-title-short} Portal) access.
 You can use the same Central instance to secure multiple {ocp} or Kubernetes clusters.
 * *Central DB*: Central DB is the database for {product-title-short} and handles all data persistence. It is currently based on PostgreSQL 13.
-* *Scanner*: Scanner is a Red Hat-developed and certified vulnerability scanner for scanning container images. Scanner performs the following functions:
+* *Scanners*: Beginning with release 4.4, {product-title-short} provides two Red Hat-developed and certified vulnerability scanners for scanning container images. Scanner V4, which was developed by consolidating the StackRox and Clair scanners, is the default scanner for language and OS-specific image components. The StackRox Scanner was the default scanner in product-title-short before Scanner V4 was introduced in release 4.4. The StackRox Scanner originates from a fork of the Clair v2 open source scanner. You must continue using this scanner for RHCOS node scanning and for active vulnerability management until that functionality is supported using Scanner V4 in a future release. The scanners perform the following functions:
```

Comment:
> certified

Scanner V4 is not certified yet, and Scanner v2 is dangerously close to losing its certification. Perhaps we should leave out the certified part?

Reply:
> consolidating the StackRox and Clair scanners

not quite true, as mentioned earlier in the review ^. I know the project name "Clair Consolidation" added to the confusion, though, so sorry 😬

Reply:
> product-title-short

Missing brackets?

Reply:
> active vulnerability management

Mentioned before, please leave this out. There are no plans to add this to Scanner V4